src/txn/txn_recover.c

```diff
@@ -717,6 +717,15 @@ __recovery_setup_file(WT_RECOVERY *r, const char *uri, const char *config)
     uint32_t fileid, lsnfile, lsnoffset;
     char lsn_str[WT_MAX_LSN_STRING];
 
+#ifdef WT_STANDALONE_BUILD
+    /*
+     * Check that we aren't opening a database with a table with an unsupported format. We only do
+     * this check for standalone as we can control whether MongoDB passes the correct format
```

Comment:
Should that be changed, or am I just not understanding the case?

```suggestion
     * this check for standalone as we cannot control whether MongoDB passes the correct format
```

Reply:
My current comment is correct, perhaps poorly worded. Basically what I am saying is that we do not need to do this check because we are in control of what storage format MongoDB uses.

Reply:
Oh, now I see, thanks!